pkg/api/apierror.go

```diff
@@ -38,10 +39,10 @@ func (a *APIError) GetCode() int {
 // Error creates a new API error from the code and error.
 func Error(code int, err error) *APIError {
 	if err == nil {
-		err = fmt.Errorf("Error pointer was nil")
+		err = errors.New("Error pointer was nil")
 	}
 
-	return New(code, err.Error())
+	return New(code, err.Error()) //nolint:govet
```

Comment:
Out of curiosity, what was the issue that `govet` had here?

Reply:
The reported issue is as follows:

```
Error: pkg/api/apierror.go:44:19: printf: non-constant format string in call to github.com/cilium/cilium/pkg/api.New (govet)
  	return New(code, err.Error())
```

I didn't see a straight-forward way to avoid that, so the nolint comment seemed like the most appropriate option.